rasa/cli/train.py

```diff
@@ -144,3 +159,30 @@ def extract_additional_arguments(args: argparse.Namespace) -> Dict:
         arguments["debug_plots"] = args.debug_plots
 
     return arguments
+
+
+def _get_valid_config(
+    config: Optional[Text],
+    mandatory_keys: List[Text],
+    default_config: Text = DEFAULT_CONFIG_PATH,
+) -> Text:
+    config = get_validated_path(config, "config", default_config)
+
+    if not os.path.exists(config):
+        print_error(
+            "The config file '{}' does not exist. Use '--config' to specify a "
+            "valid config file."
+            "".format(config)
+        )
+        exit(1)
+
+    missing_keys = missing_config_keys(config, mandatory_keys)
+    if missing_keys:
+        print_error(
+            "The config file '{}' is missing mandatory parameters: "
+            "'{}'. Add missing parameters to config file and try again."
```

Comment:
do we have a docs page we could link here?

Reply:
Not really -  the mandatory NLU parameters are discussed on one page and the mandatory Core parameters on another page.